entgql/annotation.go

```diff
@@ -514,6 +515,24 @@ func NewDirective(name string, args ...DirectiveArgument) Directive {
 	}
 }
 
+// DirectiveChildren returns a ChildValueList for the directive.
+func DirectiveChildren(k ast.ValueKind, s []string) ast.ChildValueList {
+	var list ast.ChildValueList
+
+	for _, v := range s {
+		child := &ast.ChildValue{
+			Value: &ast.Value{
+				Raw:  v,
+				Kind: k,
+			},
+		}
+
+		list = append(list, child)
+	}
+
+	return list
+}
```

Comment:
```suggestion

// DirectiveListArgument returns an argument with the list of values.
func DirectiveListArgument(name string, kind ast.ValueKind, rawValues ...string) DirectiveArgument {
	list := make(ast.ChildValueList, len(rawValues))
	for i, v := range rawValues {
		list[i] = &ast.ChildValue{
			Value: &ast.Value{
				Raw:  v,
				Kind: kind,
			},
		}
	}
	return DirectiveArgument{
		Children: list,
		Kind:     ast.ListValue,
		Name:     name,
	}
}
```

Reply:
I really liked this suggestion as it can handle cases of default values for directives (I imagine). I remember discussing with @crossworth about that. He mentioned that that would be cases where directives could have default values and I suggested using variadic functions to handle it. I think it does the job really well. Thank you for that.

What do you think @crossworth?

Reply:
If you guys are okay with the suggestion (I'm) I can accept (commit) and push another one to fix the parts that are using the old version of `DirectiveChildren` to use `DirectiveListArgument`. Just let me know.

Reply:
The end result:

```golang
func HasPermissions(permissions []string) entgql.Directive {
	return entgql.NewDirective(
		"hasPermissions",
		entgql.DirectiveListArgument("permissions", ast.StringValue, permissions...),
	)
}
```

I included the tests as well for the function.